fuuk/people/models/article.py

```diff
@@ -185,7 +185,7 @@ class Meta:
 
 class ArticleConference(Article):
     class Meta:
-        app_label = 'publications'
+#        app_label = 'publications'
```

Comment:
app_label is probably not requied. Even though I added them in last commit to be consistent with previous models.